packages/test/test-utils/src/testObjectProvider.ts

```diff
@@ -199,6 +205,9 @@ export class TestObjectProvider {
             loader,
             this.driver.createCreateNewRequest(this.documentId),
```

Comment:
Good idea! This refactoring might take longer time. My plan was to unblock tests and move on to fixing examples. We can get back to revisiting test object provider once we're done with the container ID work. 

Does it make sense?

Reply:
yep, but please make sure that this will work for all real service target (including t9s, r11s, frs, and odsp)
Please open an issue to track our plan to refactor this better.